modules/platform/nuxeo-fsexporter/src/test/java/org/nuxeo/io/fsexporter/test/TestFSExporterCollection.java

```diff
@@ -0,0 +1,102 @@
+import org.nuxeo.ecm.collections.core.test.CollectionFeature;
+import org.nuxeo.ecm.core.api.Blob;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.PathRef;
+import org.nuxeo.ecm.core.api.impl.blob.StringBlob;
+import org.nuxeo.io.fsexporter.FSExporter;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+@RunWith(FeaturesRunner.class)
+@Features(CollectionFeature.class)
+@Deploy("nuxeo-fsexporter")
+public class TestFSExporterCollection {
+
+    @Inject
+    CoreSession session;
+
+    @Inject
+    CollectionManager collectionManager;
+
+    @Inject
+    FSExporter service;
+
+    @Test
+    public void ShouldExportCollection() throws Exception {
+
+        String COLLECTION_NAME = "CollectionNameTest";
+        String COLLECTION_DESCRIPTION = "dummy";
+
+        // path : testWorkspace
+        DocumentModel testWorkspace = session.createDocumentModel("/default-domain/workspaces", "testWorkspace",
+                "Workspace");
+        testWorkspace = session.createDocument(testWorkspace);
+
+        // path : testFile
+        DocumentModel testFile = session.createDocumentModel("", "testFileName", "File");
+        Blob blobTestFile = new StringBlob("some content");
+        blobTestFile.setFilename("My_File_In_Section.txt");
+        blobTestFile.setMimeType("text/plain");
+        testFile.setPropertyValue("file:content", (Serializable) blobTestFile);
+        testFile = session.createDocument(testFile);
+
+        // path : testWorkspace/testFolder
+        DocumentModel testFolder = session.createDocumentModel(testWorkspace.getPathAsString(), "testFolderName",
+                "Folder");
+        testFolder = session.createDocument(testFolder);
+
+        // path : testWorkspace/testFolder/testFolderFile
+        DocumentModel testFolderFile = session.createDocumentModel(testFolder.getPathAsString(), "testFolderFileName",
+                "File");
+        Blob blobTestFolderFile = new StringBlob("some content");
+        blobTestFolderFile.setFilename("My_File_In_Section.txt");
+        blobTestFolderFile.setMimeType("text/plain");
+        testFolderFile.setPropertyValue("file:content", (Serializable) blobTestFolderFile);
+        testFolderFile = session.createDocument(testFolderFile);
+
+        collectionManager.addToNewCollection(COLLECTION_NAME, COLLECTION_DESCRIPTION, testWorkspace, session);
+        String COLLECTION_FULL_PATH = collectionManager.getUserDefaultCollections(session).getPathAsString() + "/"
+                + COLLECTION_NAME;
+
+        assertTrue(session.exists(new PathRef(COLLECTION_FULL_PATH)));
+        DocumentModel collection = session.getDocument(new PathRef(COLLECTION_FULL_PATH));
+
+        collectionManager.addToCollection(collection, testFile, session);
+
+        String tmp = Environment.getDefault().getTemp().getPath();
```

Comment:
In order to work correctly in the CI you should use `FeaturesRunner.getBuildDirectory()` method and then gives this folder to a `TemporaryFolder` rule such as below:
```
    @Rule
    public TemporaryFolder folder = new TemporaryFolder(new File(FeaturesRunner.getBuildDirectory()));
```
Then in your test code:
```
folder.newFolder("fs-exporter").getAbsolutePath()
```
Note: `fs-exporter` is there to add isolation between tests.